src/api/profile.api.js

```diff
@@ -0,0 +1,31 @@
+import log from '@/utils/logging'
+import axios from 'axios'
+import qs from 'qs'
+import { API_BASE_URL } from '@/config'
+
+export const profile = (data) => {
+    return new Promise((resolve, reject) => {
+        if (!data || !data.geom) {
+            const errorMessage = `Geom not provided`
+            log('error', errorMessage)
+            reject(errorMessage)
+        }
+        axios
+            .post(`${API_BASE_URL}rest/services/profile.json`, qs.stringify(data), {
```

Comment:
Have you tried initializing URLSearchParams with an object?

Reply:
We can use URLSearchParams instead of qs. I replaced it